### ISSUE TITLE
Draft an IA plan for Google Analytics and Google Tag Manager

### ISSUE THREAD

**Issue by vanitabarrett** [alphagov/govuk-design-system#1608]
## What
Draft an IA plan for use of Google Tag Manager and Google Analytics. We have a [draft plan](https://docs.google.com/document/d/1z9Y9_Kh8qDim_eqAq2XYKNS-aDRRtrSxFmBEwwSpzVk/edit) from the last time Google Tag Manager was used which we might be able to work from.

## Why
We need to make sure that Google Tag Manager is set up securely and only those who need access have access.

## Who needs to know about this
Mainly: Sara; Eoin; Vanita
The whole team probably need to be aware of the process

## Done when

The document should cover making sure:
- [x] people with access to Google Tag Manager have the correct role and access
- [x] we have a deployment workflow drafted for when we want to make changes within Google Tag Manager and make them live
- [x] we have a process for testing changes in a controlled environment
- [x] we have a process for revoking access for people leaving the team
- [x] we have new documentation covering everything that has been decided

Done when
- [x] We have a document similar to [this one](https://docs.google.com/document/d/1z9Y9_Kh8qDim_eqAq2XYKNS-aDRRtrSxFmBEwwSpzVk/edit). First step is probably reviewing that draft and deciding what we can keep vs what we need to change
- [x] revoke GTM access from anyone who no longer needs it - this has been added to a separate card
- [x] Confirm who needs to be involved in signing-off the above document
- [x] Get sign-off
- [x] Created cards for follow-up actions, e.g: creating/updating documentation; implementing leavers process etc and added them to the epic card
- [x] Added giving access to new starter Trello

**Comment by vanitabarrett** [comment]
Draft document created here: https://docs.google.com/document/d/1DqtAMuxIr2Pwm1UKVVZPYqs6Hm7M7lBGyQQI0FQKrs0/edit?usp=sharing

**Comment by vanitabarrett** [comment]
From talking to the PA community, it sounds like we just need to draft this and agree it as a team. No need for anyone else to be involved in sign-off.

**Comment by vanitabarrett** [comment]
Mostly done, the only thing left is detailing the testing within the deployment process. A bit unsure how we test changes if there are also related code changes within the repo - do we need to merge and deploy those code changes to production before we can preview the events in GTM? Will devs be able to test things locally before that point?

**Comment by vanitabarrett** [comment]
Draft deployment plan documented here: https://docs.google.com/document/d/1ABVh2noQvKZVyGkHFFzkBBgG8kQmfGuKI0bKUR8I5e0/edit#

Will be copied into the main doc once it's complete

**Comment by vanitabarrett** [comment]
Blocked until we've finalised our multi-env testing approach, more on this here: https://github.com/alphagov/govuk-design-system/issues/1616

**Comment by vanitabarrett** [comment]
Finished document here: https://docs.google.com/document/d/1DqtAMuxIr2Pwm1UKVVZPYqs6Hm7M7lBGyQQI0FQKrs0/edit#
